app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split_command = command.split(" ")
```

Comment:
There is no need to explicitly indicate that the split symbol should be space, it is a default option.